quicktests/overlaying/overlaying.js

```diff
@@ -217,10 +217,6 @@ function loadAllQuickTests(quicktestsPaths, firstQTBranch, secondQTBranch){
   quicktestsPaths.forEach(function(path) { //for each quicktest
     var name = path.replace(/\w*\/|\.js/g, "");
     d3.text("http://localhost:9999/" + path, function(error, text) {
-      if (error !== null) {
-        console.warn("Tried to load nonexistant quicktest ");
```

Comment:
`throw new Error("Tried to load nonexistant quicktest")`. Being silent while testing is not a good thing.